### ISSUE TITLE
program_id definition seems incorrect

### ISSUE THREAD

**Issue by jsiobj** [cbornet/netatmo-swagger-decl#10]
Hello,
I am using OpenHAB2, which makes use of you swagger definition, with a Netatmo Thermostat but it fails with the following error :

```
17:34:20.527` [INFO ] [me.event.ThingStatusInfoChangedEvent] - 'netatmo:NAPlug:2a0453b1' changed from ONLINE to OFFLINE: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 1880 path $.body.devices[0].therm_program..program_id
```

Digging a little bit (actually, took me ages to figure it out and some discussion with @clinique ...)

Netatmo API returns the following : 

```
"therm_program": {
          "xx:xx:xx:xx:xx:xx": {
            "zones": [
           ... 
           ],
            "timetable": [
          ...
            ],
            "program_id": {
              "$id": "aaaaaaa"
            },
            "name": "Standard"
          }
```

Changing program_id definition in swagger file from

```
                 "program_id": {
                    "type": "string"
                 }, 
```

to

```
                 "program_id": {
                    "type": "object",
                    "additionalProperties": {
                        "$id": {
                            "type": "string"
                        }
                    }
                 }, 
```

seems to solve my problem !

I made the changes here : https://github.com/jsiobj/netatmo-swagger-decl/blob/therm_prog_id/swagger.json

But I am not used to Git and as you just made some modifications, can I push a PR directly ?

Thanks.
JS

**Comment by jsiobj** [comment]
Hello @cbornet,

Did you get a chance to look at this ?

Thanks.
JS

**Comment by cbornet** [comment]
Sorry. I'll check this asap !

**Comment by cbornet** [comment]
@jsiobj What is the request that is being done ? Can you check with /getthermstate and /getthermostatsdata ? It's possible that Netatmo outputs different therm_program structure. It would be lame but in this case we would have to use different models.

**Comment by jsiobj** [comment]
Hello @cbornet
I've not been working on this for a while and just checked it again. I used your heroku app and... program_id is just a string now... for both getthermstate and getthermostatdata.
I don't know what I did but I am either becoming crazy or there has been a change in the way the API is answering.
Anyway, my openHAB instance with the Netatmo binding is now working fine. So I guess this can be closed.
Thanks & regards.
JS
